docs/multi-manager-setup.md

```diff
@@ -11,15 +11,15 @@ weight=3
 
 # High availability in Docker Swarm
```

Comment:
@abronan Can you check and make sure I add the `docker run -d swarm` appropriately?